frontends/p4/inlining.h

```diff
@@ -55,25 +55,25 @@ struct CallInfo : public IHasDbPrint {
 };
 
 class SymRenameMap {
-    std::map<const IR::IDeclaration *, cstring> internalName;
-    std::map<const IR::IDeclaration *, cstring> externalName;
+    std::map<const IR::IDeclaration *, IR::ID> internalName;
+    std::map<const IR::IDeclaration *, IR::ID> externalName;
 
  public:
     void setNewName(const IR::IDeclaration *decl, cstring name, cstring extName) {
         CHECK_NULL(decl);
         BUG_CHECK(!name.isNullOrEmpty() && !extName.isNullOrEmpty(), "Empty name");
         LOG3("setNewName " << dbp(decl) << " to " << name);
         if (internalName.find(decl) != internalName.end()) BUG("%1%: already renamed", decl);
-        internalName.emplace(decl, name);
-        externalName.emplace(decl, extName);
+        internalName.emplace(decl, IR::ID(name, decl->toString()));
```

Comment:
maybe we shuld rather do something like `decl->name.originalName` here? Using `toString` might be fragile.